e2e-tests/gef/cypress/support/commands/loadData.js

```diff
@@ -2,14 +2,12 @@
  * Mock data loader:
  * 1. Clears all the collections
  * 2. Inserts mocked data
-*/
+ */
 
 const loadData = () => {
-  // Ensure project root directory
-  cy.exec('if [ -d "dtfs2" ]; then cd dtfs2; fi');
-
   // Load mock data
-  cy.exec('npm run load');
+  // Command context is the 'dtfs2/e2e-tests/gef' directory
+  cy.exec('cd .. && npm run load');
```

Comment:
It should be just `npm run load`

Reply:
The context of this command is ‘dtfs2/e2e-tests/gef’, not ‘dtfs2/e2e-tests’, so this command is correct, and points to the new ‘load’ script I added to e2e-tests, as you requested. I added a comment above to clarify the context. The tests that use this are all passing.

Please take another look.